index.d.ts

```diff
@@ -44,6 +44,21 @@ export type Instance = {
 
 export type Unmount = () => void;
 
+/**
+ * Hook that calls inputHandler callback with input that program received.
```

Comment:
```suggestion
 * Hook that calls the `inputHandler` callback with the input that the program received.
```

Reply:
I would clarify when the inputHandler is called. It's not clear whether it's for each keypress, each line, or something else.